src/NzbDrone.Common.Test/Http/HttpHeaderFixture.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NUnit.Framework;
+using NzbDrone.Test.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NzbDrone.Common.Http;
+using System.Collections.Specialized;
+
+namespace NzbDrone.Common.Test.Http
+{
+    [TestFixture]
+    public class HttpHeaderFixture : TestBase
+    {
+        [TestCase("text/html; charset=\"utf-8\"")]
+        [TestCase("text/html; charset=utf-8")]
+        public void should_get_encoding(string contentType)
```

Comment:
`should_get_encoding_from_content_type_header` Would give it a bit more details.